### DISCUSSION TITLE
Is getTraversalObj() still working for v4.0.9 ?

### DISCUSSION THREAD

**Discussion by ningji** [NaturalIntelligence/fast-xml-parser#485] [Q&A]
Hi everyone,

We used to use getTraversalObj() for v3, when upgrading to v4.0.9, it crashes,
```
  const tObj = fastXmlParser.getTraversalObj(xmlFile, {
                             ^
TypeError: fastXmlParser.getTraversalObj is not a function.

```
1. i didn't see this clearly from change log, so looked into the source code and seems this function is gone.
2. so for 
```
var tObj = parser.getTraversalObj(xmlData,options);
var jsonObj = parser.convertToJson(tObj,options);
```

shall i just replace these 2 steps with
```
    const parser = new XMLParser(options);
    let jsonObj = parser.parse(xmlData, false);
```
    
    Thanks !

**Comment by ningji** [comment]
Just a quick update, seems i was able to replace it with XMLParser now, will keep you posted for the next few days, thx !

**Reply by amitguptagwl** [reply]
No, `getTraversalObj()` is not supported anymore. Instead you can use parser to parse XML maintaining the sequence of tags or pretty compressed output.

**Reply by miguel-padilla92** [reply]
can you let me know how you changed getTraversalObj()? I'm trying to figure it out but I'm not able to do it yet
